slave.mk

```diff
@@ -939,6 +939,7 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
                 $(LIBNSS_TACPLUS) \
                 $(MONIT) \
                 $(OPENSSH_SERVER) \
+		$(SMARTMONTOOLS) \
```

Comment:
@alexrallen please remove this change and raise a new PR apart from this one. Still have a doubts that Community will accept it...

Reply:
I talked with the MSFT maintainer and they were okay with this. Was concerned as well. 

I can raise it separately.